server/src/main/resources/db/common/V119__alter_pulse_response_team_id.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pulse_response DROP CONSTRAINT pulse_response_teamMemberId_fkey;
```

Comment:
It seems odd to me that we would need to drop a foreign key reference to allow null. Is this in some way related to: https://stackoverflow.com/questions/23325838/postgresql-null-value-in-foreign-key-column ?

Reply:
@mkimberlin Honestly, I just assumed that because it was referencing the member_profile, that it could not be null.  I did not test without this.

Reply:
I removed that file, it was not necessary and took out the history.go(0) call too.